spring-cloud-kubernetes-commons/src/main/java/org/springframework/cloud/kubernetes/commons/config/reload/ConfigReloadAutoConfiguration.java

```diff
@@ -64,22 +64,20 @@ public TaskSchedulerWrapper<TaskScheduler> taskScheduler() {
 	public ConfigurationUpdateStrategy configurationUpdateStrategy(ConfigReloadProperties properties,
 			ConfigurableApplicationContext ctx, Optional<RestartEndpoint> restarter, ContextRefresher refresher) {
 		String strategyName = properties.getStrategy().name();
-		switch (properties.getStrategy()) {
```

Comment:
this simply changes to the new way switch statements work in java-17. This PR must wait for the other one in review - because it fixes a test failure